vhdbuilder/packer/components.json

```diff
@@ -110,6 +110,7 @@
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/apiserver-network-proxy/agent:*",
       "versions": [
+        "v0.0.17",
         "v0.0.16",
         "v0.0.15",
```

Comment:
shall we delete the old ones? Only v0.0.16 is in use, and we can remove the rest.

Reply:
good call. Will confirm and address in a separate PR.